### ISSUE TITLE
Broken create command on CLI (same id)

### ISSUE THREAD

**Issue by orl0pl** [orl0pl/box-master#2]
```
 > create
create
Enter the attributes JSON of the new box: {"name": "Fork"}
{"name": "Fork"}
Enter the ID of the parent box, or leave 
blank to make it a root box: 0
0
Created box with attributes "[object Object]".

 > tree
tree
|-- HE (0)
  |-- WTF (3)
  |-- Fork (3)
|-- 2 (1)
  |-- WTF (3)
  |-- Fork (3)
|-- Fork (3)
```

I need to fix this quickly.

**Comment by orl0pl** [comment]
Issue seems to appears after restarting the CLI.

**Comment by orl0pl** [comment]
Issue fixed with commit 1d0cc74
